boa_parser/src/parser/expression/update.rs

```diff
@@ -56,6 +56,21 @@ impl UpdateExpression {
     }
 }
 
+#[inline]
```

Comment:
```suggestion
```

The compiler ignores the `#[inline]` annotation on functions that aren't public.